### ISSUE TITLE
Wuhan, Hubei, China

### ISSUE THREAD

**Issue by justinelliotmeyers** [openaddresses/openaddresses#2387]
looks like a start ;)
http://www.digitalwuhan.gov.cn:8399/arcgis/rest/services/HHLJ/hhljdy/MapServer/0

![wuhan](https://cloud.githubusercontent.com/assets/5324316/22156181/2c599cfc-df00-11e6-8fa3-0e941f61c213.jpg)

**Comment by trescube** [comment]
I have no idea how to parse this source.  Would need help from someone familiar with addressing in China.

**Comment by trescube** [comment]
@thatdatabaseguy you seem to know all sorts of random things... care to take a crack at this one?

**Comment by trescube** [comment]
@thatdatabaseguy Sorry, I didn't see that you had already done this source.  Could you reference this issue in your upcoming PR?

**Comment by thatdatabaseguy** [comment]
Indeed! However, this is the one where there are ~3k addresses but pyesridump is only pulling 1k. Got it to work with a few changes to pyesridump though.

The [other Hubei data source](https://github.com/openaddresses/openaddresses/issues/2446) has no delineated fields and it's not clear that those addresses are always well-structured for parsing with regex (could be "Near Development Road 317", etc.)
